CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta15 (in progress)
+==========================
+
+New Features
+------------
+
+- twilio-video.js will now support the Unified Plan SDP format for Google Chrome. Google Chrome
```

Comment:
will now support => will now by default support

Reply:
@makarandp0 Chrome is the one with default support for Unified Plan after version 72. The SDK supports whatever SDP format supported by the browser.